build-system/compile/post-closure-babel.js

```diff
@@ -86,22 +86,23 @@ exports.postClosureBabel = function () {
       file.contents,
       file.sourceMap
     );
-    const {compressed, terserMap} = terserMinify(code);
 
-    file.contents = Buffer.from(compressed, 'utf-8');
-    file.sourceMap = remapping(
-      [terserMap, babelMap, map],
-      () => null,
-      !argv.full_sourcemaps
-    );
+    terserMinify(code).then(({compressed, terserMap}) => {
```

Comment:
Should there be a `catch` handler, too?

Reply:
You're absolutely right. Not sure why I didn't add one.

Reply:
Will add an additional exception handler in a future PR.